.buildkite/scripts/send_and_wait_for_test_bag.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python
+"""
+Send a test bag to the storage service, then wait for it to be registered
+and successfully stored.
+"""
+
+import sys
+import time
+
+import boto3
+
+from wellcome_storage_service import RequestsOAuthStorageServiceClient
+
+
+def get_secret_string(sess, *, secret_id):
+    client = sess.client("secretsmanager")
+    return client.get_secret_value(SecretId=secret_id)["SecretString"]
+
+
+def get_client(*, api_url):
+    sess = boto3.Session()
+
+    client_id = get_secret_string(sess, secret_id="buildkite/storage_service/client_id")
+    client_secret = get_secret_string(
+        sess, secret_id="buildkite/storage_service/client_secret"
+    )
+
+    return RequestsOAuthStorageServiceClient(
+        api_url=api_url,
+        client_id=client_id,
+        client_secret=client_secret,
+        token_url="https://auth.wellcomecollection.org/oauth2/token",
+    )
+
+
+if __name__ == "__main__":
```

Comment:
I think this will keep a CI instance awake and running for a long(ish) period. I don't think that is necessarily a problem, but would be good to quantify.

Reply:
Yeah, it's not dissimilar to what we do waiting for weco-deploy deployments.

I think we should look at running agents on differently sized instances, a la https://buildkite.com/docs/agent/v3/elastic-ci-aws#multiple-instances-of-the-stack